src/main/scala/com/firfi/slackbaka/workers/NomadWorker.scala

```diff
@@ -112,7 +112,7 @@ class NomadWorker(responder: ActorRef) extends BakaRespondingWorker(responder) w
   def request(placeName: PlaceName): Future[HttpResponse] = {
     import com.netaporter.uri.dsl._
     Source.single(RequestBuilding.Get(
-      ("/search?" ? ("q" -> placeName.name)
```

Comment:
This change actually not doing anything. If you want to look into case sensitivity, please check comparison/validation done further in workflow.